models/image.go

```diff
@@ -161,18 +146,6 @@ func (d *Download) Validate() error {
 	return nil
 }
 
-// Refresh regenerates the values for public and href according to the download state and the provided values
```

Comment:
NB. the functionality being performed by Refresh should instead live within the calling service. Ie. the importer or the static file publisher will compute the HREF and it will pass this in to the API via the PUT endpoints. This has been discussed with James